src/ui/react/src/adapter/alloy-editor.js

```diff
@@ -43,6 +45,12 @@
 
             editor.config.selectionKeystrokes = this.get('selectionKeystrokes');
 
+            if (config.extraPlugins) {
```

Comment:
Hey @antoniopol06, I think this is not 100% right. This will always add the drag_resize plugin to the `extraPlugins` property, even if the user has decided to have a different set of plugins.

We should add `ae_dragresize` to the default value and simply replace the plugin with the ie version here in case it appears in the property on initialization.